tests/queries/shell_config.sh

```diff
@@ -39,10 +39,13 @@ export CLICKHOUSE_LOCAL=${CLICKHOUSE_LOCAL:="${CLICKHOUSE_BINARY}-local"}
 [ -x "${CLICKHOUSE_BINARY}-server" ] && CLICKHOUSE_SERVER_BINARY=${CLICKHOUSE_SERVER_BINARY:="${CLICKHOUSE_BINARY}-server"}
 [ -x "${CLICKHOUSE_BINARY}" ] && CLICKHOUSE_SERVER_BINARY=${CLICKHOUSE_SERVER_BINARY:="${CLICKHOUSE_BINARY} server"}
 export CLICKHOUSE_SERVER_BINARY=${CLICKHOUSE_SERVER_BINARY:="${CLICKHOUSE_BINARY}-server"}
+# benchmark
+[ -x "${CLICKHOUSE_BINARY}-benchmark" ] && CLICKHOUSE_BENCHMARK_BINARY=${CLICKHOUSE_BENCHMARK_BINARY:="${CLICKHOUSE_BINARY}-benchmark"}
```

Comment:
```suggestion
[ -x "${CLICKHOUSE_BINARY}-benchmark" ] && CLICKHOUSE_BENCHMARK_BINARY=${CLICKHOUSE_BENCHMARK_BINARY:="${CLICKHOUSE_BINARY}-benchmark ${CLICKHOUSE_BENCHMARK_OPT0:-}"}
```

Reply:
Hmmmm, maybe in that case ditch `CLICKHOUSE_BENCHMARK_BINARY` completely and assign `CLICKHOUSE_BENCHMARK` already here?

IMHO having `CLICKHOUSE_BENCHMARK_OPT0` for `_BINARY` is a bit inconsistent at least with `CLICKHOUSE_CLIENT`, where `CLICKHOUSE_CLIENT_BINARY` is just a path to the binary itself, and `CLICKHOUSE_CLIENT` contains all the needed options.

Reply:
Yep, makes sense. We just need to make sure `CLICKHOUSE_BENCHMARK` contains the options

Reply:
Ah yeah, I overlooked that it added the options *only*  if `CLICKHOUSE_BENCHMARK_BINARY` hadn't been set earlier. Thanks, I'll fix it!